### PR TITLE
Backblast - Fix damage disabled units receiving damage from backblast

### PR DESCRIPTION
**Pull request by diwako**
**When merged this pull request will:**
- Will not damage units from over pressure if damage is disabled

Fix for #7429

### CODE REVIEW

File: addons/overpressure/functions/fnc_overpressureDamage.sqf [hunk 1]
```diff
@@ -56,12 +56,13 @@ TRACE_3("cache",_overpressureAngle,_overpressureRange,_overpressureDamage);
...
             if (_x == ACE_player) then {[_damage * 100] call BIS_fnc_bloodEffect};
 
-            if (isClass (configFile >> "CfgPatches" >> "ACE_Medical")) then {
-                [_x, _damage, "body", "backblast", _firer] call EFUNC(medical,addDamageToUnit);
-            } else {
-                TRACE_1("",isDamageAllowed _x);
-                if (!isDamageAllowed _x) exitWith {}; // Skip damage if not allowed
-                _x setDamage (damage _x + _damage);
+            TRACE_1("",isDamageAllowed _x);
+            if (isDamageAllowed _x) then {
```
Comment (jokoho48) [inline addons/overpressure/functions/fnc_overpressureDamage.sqf]: I would also try to catch the ACE Medial variable
```suggestion
            if (isDamageAllowed _unit && {_unit getVariable [QEGVAR(medical,allowDamage), true]}) then {
```

Comment (commy2) [reply addons/overpressure/functions/fnc_overpressureDamage.sqf]: That variable is never read by the game anywhere else as far as I can tell.

It is only set here:
https://github.com/acemod/ACE3/blob/master/addons/spectator/functions/fnc_setSpectator.sqf#L107
but that also should be deleted, as it does nothing.

What even is the point of that variable?

Comment (commy2) [reply addons/overpressure/functions/fnc_overpressureDamage.sqf]: _unit -> _x

Comment (jokoho48) [reply addons/overpressure/functions/fnc_overpressureDamage.sqf]: ups copy-pasted it from somewhere else :P

**Pull request merged**